Assets/AltTester/AltDriver/Commands/FindObjects/AltWaitForComponentProperty.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Threading;
+using AltTester.AltDriver.Logging;
+
+namespace AltTester.AltDriver.Commands
+{
+    public class AltWaitForComponentProperty<T> : AltBaseFindObjects
+    {
+        readonly NLog.Logger logger = DriverLogManager.Instance.GetCurrentClassLogger();
+        AltObject altObject;
+        T propertyFound;
```

Comment:
There is no need to define `propertyFound` as an attribute, it can be a local variable on the `Execute` method.